.goreleaser.yml

```diff
@@ -87,6 +87,16 @@ release:
 
     For key updates, see the [changelog](https://golangci-lint.run/product/changelog/#{{ .Major }}{{ .Minor }}{{ .Patch }}).
 
+signs:
```

Comment:
Local `goreleaser release` runs should likely be done with `--skip sign` in order to not break.